src/textarea/interfaces.ts

```diff
@@ -23,6 +30,8 @@ export interface TextareaProps
    * that checks for spelling/grammar errors.
    * If you don't set it, the behavior follows the default behavior
    * of the user's browser.
+   *
+   * @deprecated Use the `spellcheck` property instead.
```

Comment:
How does it work with the documenter? Because I don't see the snapshot updated, will it show on the website api page?